src/main/kotlin/screeps/api/PathFinder.kt

```diff
@@ -32,7 +32,7 @@ external object PathFinder {
 }
 
 external interface SearchOptions : Options {
-    var roomCallback: ((String) -> PathFinder.CostMatrix)? get() = definedExternally; set(value) = definedExternally
+    var roomCallback: ((String) -> PathFinder.CostMatrix?)? get() = definedExternally; set(value) = definedExternally
```

Comment:
The docs (https://docs.screeps.com/api/#PathFinder) says we may return `false` instead of a `PathFinder.CostMatrix`. I' am not sure what happens if we just return `null`, presumably bad things.

It would be nice if you could test whether returning `null` has the same effect as `false`.
If we need to return `false` we can introduce a constant that shares an `external interface` with CostMatrix and use the new interface as the return type for this function. e.g.

```
external interface CostMatrix

val COSTMATRIX_FALSE : CostMatrix = false.unsafeCast<CostMatrix>()
```

Reply:
https://github.com/screeps/driver/blob/f26a6e5f252fa4fce2419b2edebce978d8810796/lib/path-finder.js#L123

I guess it's better to be safe than sorry. I'm not sure what happens if both those if's are false.